python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -188,11 +191,11 @@ def makeIdFactory(self, dataRef):
                              (e.g. visit, raft, sensor for LSST data).
         """
         expBits = dataRef.get("ccdExposureId_bits")
-        expId = long(dataRef.get("ccdExposureId"))
+        expId = int(dataRef.get("ccdExposureId"))
```

Comment:
Well, then in Python 2 every time you used `int` you would get a `long`. That might also not be desirable so it's not surprising it's not on by default. I think I prefer the `past.builtins` approach even though you end up with code that looks weird on python 3.